SmartWebDriver/WebBrowser.cs

```diff
@@ -742,6 +742,34 @@ public void RightClick(IWebElement webElement, string description)
             }
         }
 
+        public void DragAndDrop(PageElement elementToDrag, PageElement targetElement)
```

Comment:
Please add a summary description to this method

Reply:
I won't lie, I am new to GitHub. For adding the summary description, do I need to commit that summary and create a new pull request, or can I update this one?

Reply:
Nice, thanks for that. This is now good to merge, I unfortunately no longer have Write access, so I'm not able to at the moment, I'm hoping to get permission back. But in the meantime, if you have write access, feel free to merge!

Reply:
Looks like I do not have write access. Do you know who I should contact to get this merged in?